src/main/java/homer/model/actuator/AbstractActuatedDevice.java

```diff
@@ -12,20 +13,23 @@
 public class AbstractActuatedDevice implements ActuatedDevice {
 
     private final Actuator actuator;
+    private final Optional<String> type;
 
     /**
      * Creates a {@link AbstractActuatedDevice} whose position is controlled by an
      * {@link Actuator}.
      * 
      * @param actuator The {@link Actuator} controlling the device.
+     * @param type
      */
-    public AbstractActuatedDevice(final Actuator actuator) {
+    public AbstractActuatedDevice(final Actuator actuator, final Optional<String> type) {
```

Comment:
Optionals in arguments make no sense, but whatever.

Reply:
will fix